ldoc.lua

```diff
@@ -7,7 +7,7 @@
 --
 -- C/C++ support for Lua extensions is provided.
 --
--- Available from LuaRocks as 'ldoc' and as a [Zip file](http://stevedonovan.github.com/files/ldoc-1.4.3.zip)
+-- Available from LuaRocks as 'ldoc' and as a [Zip file](http://stevedonovan.github.io/files/ldoc-1.4.6.zip)
```

Comment:
that should also become "lunarmodules", instead of Steve's

Reply:
> that should also become "lunarmodules", instead of Steve's

Done.

Reply:
As of yet we haven't republished the docs in the new org scope.